src/components/Identity/processIdSyncs.js

```diff
@@ -35,54 +38,60 @@ const setControlObject = obj => {
 
   Object.keys(obj).forEach(id => arr.push(`${id}-${obj[id]}`));
 
-  cookie.set(`${namespace}idSyncControl`, arr.join("_"), {
-    expires: 6 * 30 // 6 months
-  });
+  alloyCookie.set(ID_SYNC_CONTROL, arr.join("_"));
 };
 
-export default ({ destinations, config, logger }) => {
-  if (config.idSyncsEnabled) {
-    const controlObject = getControlObject();
-    const now = new Date().getTime() / millisecondsPerHour; // hours
+export default ({ destinations, config, logger, cookie }) => {
+  alloyCookie = cookie;
 
-    Object.keys(controlObject).forEach(key => {
-      if (controlObject[key] < now) {
-        delete controlObject[key];
-      }
-    });
+  if (!config.idSyncsEnabled) {
+    return;
+  }
 
-    const idSyncs = destinations
-      .filter(
-        dest => dest.type === "url" && controlObject[dest.id] === undefined
-      )
-      .map(dest =>
-        assign(
-          {
-            id: dest.id
-          },
-          dest.spec
-        )
-      );
+  const controlObject = getControlObject();
+  const now = new Date().getTime() / MILLISECONDS_PER_HOUR; // hours
 
-    if (idSyncs.length) {
-      fireDestinations({
-        logger,
-        destinations: idSyncs
-      }).then(result => {
-        const timeStamp = Math.round(
-          new Date().getTime() / millisecondsPerHour
-        ); // hours
+  Object.keys(controlObject).forEach(key => {
+    if (controlObject[key] < now) {
+      delete controlObject[key];
+    }
+  });
 
-        result.succeeded.forEach(idSync => {
-          const ttl = (idSync.ttl || 7) * 24; // hours
+  const idSyncs = destinations
+    .filter(dest => dest.type === "url" && controlObject[dest.id] === undefined)
+    .map(dest =>
+      assign(
+        {
+          id: dest.id
+        },
+        dest.spec
+      )
+    );
+
+  if (idSyncs.length) {
+    fireDestinations({
+      logger,
+      destinations: idSyncs
+    }).then(result => {
+      const timeStamp = Math.round(
+        new Date().getTime() / MILLISECONDS_PER_HOUR
+      ); // hours
+
+      result.succeeded.forEach(idSync => {
+        const ttl = (idSync.ttl || 7) * 24; // hours
+
+        if (idSync.id !== undefined) {
+          controlObject[idSync.id] = timeStamp + ttl;
```

Comment:
In one of the recent standups, we decided that when saving numbers into cookies from our various components, we were going to use a base radix of 36 for optimized storage. In other words, take the result of `timeStamp + ttl` and call `toString(36)` and then when you use the timestamp from the cookie, call `parseInt(myNumber, 36)`.